Resources/public/js/views/ez-languageswitcherview.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-languageswitcherview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Language Switcher View class
+     *
+     * @module ez-languageswitcherview
+     */
+    Y.namespace('eZ');
+
+    var EXPANDED_LANGUAGE_LIST = 'is-expanded';
+
+    /**
+     * The raw content view
+     *
+     * @namespace eZ
+     * @class LanguageSwitcherView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.LanguageSwitcherView = Y.Base.create('languageSwitcherView', Y.eZ.TemplateBasedView, [Y.eZ.AccordionElement], {
```

Comment:
for the language switcher, I would rather use the Y.eZ.Expandable extension instead, that way you already get the logic of adding/removing the `is-expanded` class and given that the additional languages list will be absolutely positioned we don't need the maths done in accordion element to achieve a nice transition.